api/test/programs/test-waitlist/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "test-waitlist"
+version = "0.1.0"
+authors = ["Gear Technologies"]
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib"]
```

Comment:
```suggestion
```

As we discussed, you can now use `gear_wasm_builder`

Reply:
Only in case of building through gear builder?

Reply:
> Only in case of building through gear builder?

Yeah